geometry_msgs/Polygon fly_zone
geometry_msgs/Polygon[] no_fly_zones

uint8 number_of_drones

float64 sweeping_step
float64 distance_for_turning
int32 max_number_of_extra_points

uint16 drones_altitude
float64 unique_altitude_step
float64 start_lat
float64 start_lon
float64 decomposition_rotation
float64 max_polygon_area

uint8 rotations_per_cell
uint16 no_improvement_cycles_before_stop

bool override_battery_model
float64 battery_cell_capacity
uint8 battery_number_of_cells

bool override_drone_parameters
float32 drone_mass
float32 drone_area
float32 average_acceleration
float32 propeller_radius
uint8 number_of_propellers
---
bool success
string message
mrs_msgs/Path[] paths_gps
float64[] energy_consumptions
